wiki/Ranking_Criteria/osu!catch/en.md

```diff
@@ -28,6 +28,7 @@ These terms are frequently appearing when getting in touch with osu!catch diffic
 -   **Hyperjump:** A more exaggerated spacing which cannot be caught by normal dashing. During play, hyperdash will be triggered between the two objects, characterised by a coloured outline on the first object.
 -   **Edge Dash:** A very large spacing between two objects where the required Trigger Distance is not reached, and as such, a hyperjump is not generated. The first object must be caught with the edge of the plate in order to catch the second object at all.
 -   **Trigger distance:** The minimum spacing between two objects at which a hyperdash is generated between them.
+-   **Equivalent BPM:** A BPM value in which a certain snap is time-equivalent (same duration in ms) to a reference BPM/snap pair. This is calculated as Equivalent BPM = (Reference BPM)*(Desired Snap)/(Reference Snap).
```

Comment:
What is a snap pair? Which BPM reference? Generally I read that 5 times and I still don't know what it is.
Also make the equation in code blocks (\`text\`)

Reply:
It's meant to be a pair "reference BPM/reference snap". Changing the wording to make it more clear